src/openrct2/world/sprite.c

```diff
@@ -30,10 +30,12 @@
 uint16 gSpriteListHead[6];
 uint16 gSpriteListCount[6];
 static rct_sprite _spriteList[MAX_SPRITES];
+static bool _spriteFlashingList[MAX_SPRITES];
 #else
 uint16 *gSpriteListHead = RCT2_ADDRESS(RCT2_ADDRESS_SPRITE_LISTS_HEAD, uint16);
 uint16 *gSpriteListCount = RCT2_ADDRESS(RCT2_ADDRESS_SPRITE_LISTS_COUNT, uint16);
 static rct_sprite *_spriteList = RCT2_ADDRESS(RCT2_ADDRESS_SPRITE_LIST, rct_sprite);
+static bool _spriteFlashingList = RCT2_ADDRESS(RCT2_ADDRESS_SPRITE_LISTS_COUNT, bool);
```

Comment:
You probably shouldn't modify the with-RCT2 builds and this certainly does not look correct.